2024/day01/solution.md

```
@@ -0,0 +1,49 @@
+# What is DevOps ? 🤔
+### Imagine a world where software development and IT operations work together in harmony, like friends working together to solve a puzzle. That's the vision of DevOps. It's a cultural shift that aims to break down silos, improve collaboration between teams and automation which leads to faster time-to-market.
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)